src/pyggdrasil/interface/__init__.py

```diff
@@ -76,3 +80,71 @@ def append(self, iteration: int, tree: TreeNode, log_probability: float):
         self.iterations = jnp.append(self.iterations, iteration)
         self.trees.append(tree)
         self.log_probabilities = jnp.append(self.log_probabilities, log_probability)
+
+
+@dataclass
+class AugmentedMcmcData(PureMcmcData):
+    """Distance/Similarity and True Tree boolean enhanced MCMC data
+
+    intended to be human-readable and easy to use for plotting"""
```

Comment:
This class is not used currently. I tried to come up with a good format to save all MCMC data and analytics, excluding trees. Frankly, a pandas data frame might have been easier. I am planning to write functions to convert from `PureMcmcData` to this `AugmentenMcmcData`, by making simple choices for which Distances to use and then save this to disk as a csv or dataframe. 

Once that works, I'll refactor the visualization scripts to read in these files - to separate the analytics from the visualisation. Perhaps even creating a new submodule `analyze` ? 

What do you think ? I am not very convinced by my design of this class.